database/migrations/2014_10_12_000000_create_users_table.php

```diff
@@ -14,10 +14,8 @@ public function up()
     {
         Schema::create('users', function (Blueprint $table) {
             $table->increments('id');
```

Comment:
okie doke. I'll remove it. Should I add a `phoenix_id` at all? Would that help when signups from Phoenix for competitions are hitting Gladiator and bringing with it data from Phoenix?